pkg/resourcesets/collector.go

```diff
@@ -436,17 +436,21 @@ func writeToBackup(ctx context.Context, resource map[string]interface{}, backupP
 	if err != nil {
 		return fmt.Errorf("error converting resource to JSON: %v", err)
 	}
-	if transformer != nil && !util.IsDefaultEncryptionTransformer(transformer) {
```

Comment:
...it technically does similar to what this intended to do, just after `transformer.TransformToStorage` is called instead of before. Obviously before was better, but no longer an option (for) now.